src/image.cpp

```diff
@@ -1331,7 +1331,16 @@ bool save_image(const surface& surf, const std::string& filename)
 		return false;
 	}
 
-	if(!filesystem::ends_with(filename, ".bmp")) {
+#if SDL_IMAGE_VERSION_ATLEAST(2, 0, 2)
+	if(filesystem::ends_with(filename, ".jpg")) {
+		LOG_DP << "Writing a png image to " << filename << std::endl;
```

Comment:
Copy-paste mistake. This line should say that it's writing a JPEG image.

(BTW, I wonder why these log messages are writing acronyms in lowercase.)